message_adapter/message_adapter.py

```diff
@@ -367,10 +390,10 @@ def __assignJsonPathValue(self, message, jspath, value):
                 dictPath += "['" + path + "']"
                 if keyNotFound or path not in currentItem:
                     keyNotFound = True
-                    exec ("message" + dictPath + " = {}")
+                    exec("message" + dictPath + " = {}")
                 currentItem = eval("message" + dictPath)
```

Comment:
I didn't realize the CMA used `eval`. That's an unsafe and bad practice as I understand it. I think we should file a follow-up ticket to look at alternative implementations.

Reply:
Agreed, this is .... not really acceptable.   I have a note to refactor or write that ticket already.